src/main/java/Calculator.java

```diff
@@ -0,0 +1,50 @@
+import java.util.ArrayList;
+import java.util.Locale;
+
+public class Calculator {
+    int guestCount;
+    ArrayList<Product> goods;
+
+    Calculator(int guestCountInit) {
+        this.guestCount = guestCountInit;
+        this.goods = new ArrayList<Product>();
+    }
+
+    public void addProduct(Product product) {
+        goods.add(product);
+    }
+
+    public static String getCurrencyName(float qty) {
```

Comment:
🍏 Рекомендую не экономить на буквах, чтобы код был более читаемым:
```suggestion
    public static String getCurrencyName(float quantity) {
```